doc/source/user_guide/io.rst

```diff
@@ -3933,6 +3934,29 @@ returned, this is equivalent to passing a
 
    store.select('df', "columns=['A', 'B']")
 
+Using level name when querying MultiIndex DataFrame.
+
+.. ipython:: python
+
+   store.select('df_mi', "foo=baz and bar=two")
+
+Using ``level_0`` and ``level_1`` for MultiIndex without name.
```

Comment:
give a statement about how the level names of ``level_i`` for the index of the level are available automatically if the level name is None